docs/configuration/configuration.md

```diff
@@ -3227,6 +3196,25 @@ There is a list of
 [integrations](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage)
 with this feature.
 
+### `<tsdb>`
+
+`tsdb` lets you config the runtime reloadable configuration of the TSDB.
+
+NOTE: Out of order ingestion is an experimental feature, but you do not need any additional flag to enable it. Setting `out_of_order_time_window` to a positive number enables it.
+
+```yaml
+# Configures how old an out-of-order/out-of-bounds sample can be w.r.t. the TSDB max time.
+# An out-of-order/out-of-bounds sample is ingested into TSDB as long as the timestamp
+# of the sample is >= TSDB.MaxTime-out_of_order_time_window.
+#
+# When out_of_order_time_window is >0, the errors out-of-order and out-of-bounds are
+# combined into a single error called 'too-old'; a sample is either (a) ingestible
+# into TSDB, i.e. it is an in-order sample or an out-of-order/out-of-bound sample
+# that is after TSDB.MaxTime-out_of_order_time_window, or (b) too-old, i.e. not in-order
+# and before TSDB.MaxTime-out_of_order_time_window.
```

Comment:
Replacing "TSDB.MaxTime-out_of_order_time_window" with "out-of-order window" (which I think is what that is) might be more clear

```suggestion
# that is within the out-of-order window, or (b) too-old, i.e. not in-order
# and before the out-of-order window.
```